packages/hooks/src/useP2PAdvertInfo.ts

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useFetch } from '@deriv/api';
+import { TSocketRequestQueryOptions } from '@deriv/api/types';
+
+/**
+ * This custom hook returns the advert info for a specific advert by calling 'p2p_advert_info' endpoint
+ */
+const useP2PAdvertInfo = (id: string, options: TSocketRequestQueryOptions<'p2p_advert_info'>) => {
+    const { data, ...rest } = useFetch('p2p_advert_info', { payload: { id }, options });
+
+    const modified_data = React.useMemo(() => {
```

Comment:
Do we need to add modified_data here? 
You are just returning the data and didn't modify anything here :thinking:

Reply:
Maybe you can remove this extra function :thinking: 

cc: @farzin-deriv

Reply:
It's okay to keep it, Later we can add more stuff here. but better to follow the same convention as other hooks 🙇🏻